libs/platform/src/lib/components/form/combobox/combobox/combobox.component.ts

```diff
@@ -58,9 +58,17 @@ export class ComboboxComponent extends BaseCombobox implements OnInit, AfterView
     /** @hidden */
     selected?: OptionItem;
```

Comment:
If this is hidden it should be prefixed with `_`, then there will be duplication with the prop on line 64